src/lib/datasource/InMemoryDatasource.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 const Datasource = require('../Datasource');
 
-class InMemoryDatasource extends Datasource {
+
+const undefinedToNull = (value) => 
```

Comment:
this function is copy-pasted.
Consider to extract it to a separate file under /utils folder

Reply:
resolved